src/main/kotlin/uk/co/poolefoundries/baldinggate/core/CoreComponents.kt

```diff
@@ -72,6 +73,21 @@ data class PositionComponent(val x: Int, val y: Int) : Component {
 
 }
 
+data class AnimationComponent(val entity: Entity, var positions: List<PositionComponent> = listOf<PositionComponent>(), var progress: Float = 0F, var isFinished: Boolean = false) : Component {
```

Comment:
Why is the positions list defaulted to an empty list? What's the purpose of an anmation with no steps in it? We might also want to name this "MoveAnimation" as we might want other types of animations (fade out for death maybe?)

Also I'm not sure this needs to be a component. I would do something like `Entity.get(VisualComponent::java.class).addAnimation(...)` and do this in the render component (or something).